package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@particle/firmware-protobuf",
+  "version": "1.0.0",
+  "description": "Particle Device OS Protobuf Definitions",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "mkdir -p dist && pbjs control/*.proto -t static-module -o dist/index.js --no-delimited --no-convert --no-verify",
```

Comment:
`mkdir -p` won't work on macOS. I suggest adding [mkdirp](https://www.npmjs.com/package/mkdirp) as dev dependency and using it instead.

Reply:
yep, and also in general prefer `fs-extra` ([link](https://github.com/jprichardson/node-fs-extra)) as it collects `mkdirp` and similar utilities without bloating things

Reply:
So it's still an issue? I've verified that `mkdir -p` works in the default shell on my wife's Mac, but I vaguely remember that it wasn't always the case.

Reply:
yeah that rings a tiny bell for me as well. that said, `man mkdir` on my mac ("Monterey" `v12.0.1`) shows the `-p` flag docs and this is internal tooling so i think we're good :+1: